integration_tests/suite/helpers/database.py

```diff
@@ -124,6 +125,33 @@ def insert_user_line_extension(self, **kwargs):
             }
             return user, line, extension
 
+    def insert_extra_user_line(self, user_id, **kwargs):
+        context = kwargs.get('context', 'foocontext')
+
+        with self.inserter() as inserter:
+            line = inserter.add_line(
+                context=context,
+                name=kwargs.get('name_line', ''.join(random.choice('0123456789ABCDEF') for _ in range(6))),
+                device=kwargs.get('device', 1),
+                commented=kwargs.get('commented_line', 0),
+                endpoint_sip_uuid=kwargs.get('endpoint_sip_uuid', None),
+                endpoint_sccp_id=kwargs.get('endpoint_sccp_id', None),
+                endpoint_custom_id=kwargs.get('endpoint_custom_id', None)
+            )
```

Comment:
```suggestion
            line_args = {}
            if 'line_name' in kwargs:
                line_args['name'] = kwargs['line_name']
            if 'device' in kwargs:
                line_args['device'] = kwargs['device']
            if 'line_commented' in kwargs:
                line_args['commented'] = kwargs['line_commented']
            if 'endpoint_sip_uuid' in kwargs:
                line_args['endpoint_sip_uuid'] = kwargs['endpoint_sip_uuid']
            if 'endpoint_sccp_uuid' in kwargs:
                line_args['endpoint_sccp_uuid'] = kwargs['endpoint_sccp_uuid']
            if 'endpoint_custom_uuid' in kwargs:
                line_args['endpoint_custom_uuid'] = kwargs['endpoint_custom_uuid']
            line = inserter.add_line(**line_args)            )
```

Or little less explicit/verbose
```suggestion
            line_args = {
                'endpoint_sip_uuid': kwargs.get('endpoint_sip_uuid'),
                'endpoint_sccp_uuid': kwargs.get('endpoint_sccp_uuid'),
                'endpoint_custom_uuid': kwargs.get('endpoint_custom_uuid'),
            }
            if 'line_name' in kwargs:
                line_args['name'] = kwargs['line_name']
            if 'device' in kwargs:
                line_args['device'] = kwargs['device']
            if 'line_commented' in kwargs:
                line_args['commented'] = kwargs['line_commented']
            line = inserter.add_line(**line_args)            )
```

* We should avoid to define multiple default value to different layer if not necessary
* I have a small preference to use `line_` prefix instead of suffix to define line parameters. IMHO it's more readable/parsing with eyes

Reply:
Yeah, it's a weird way to do it. That's how all the code in the `inserter` from `xivo_dao` is done. This method is essentially `test_dao.add_user_line_with_queue_member`, but simplified without the user creation. Was adding it here the right thing to do? I'm all up for improving it, I just kept the same names for uniformity.

Reply:
Ah ok I just saw what you based it on. Well, do you best :smile: